test/jdk/java/lang/ProcessBuilder/Basic.java

```diff
@@ -1875,7 +1888,7 @@ public void doIt(Map<String,String> environ) {
             }
             Process p = Runtime.getRuntime().exec(cmdp, envp);
             String expected = Windows.is() ? "=C:=\\,=ExitValue=3,SystemRoot="+systemRoot+"," : "=C:=\\,";
-            expected = AIX.is() ? expected + "LIBPATH="+libpath+",": expected;
+            expected = AIX.is() ? expected + "LIBPATH="+removeTestNativepathString(libpath)+",": expected;
```

Comment:
Can line 1878 (and the fix) be moved into the `removeAixExpectedVars` method.
```
1883:            if (AIX.is()) {
1884:                commandOutput = removeAixExpectedVars(commandOutput);
1885:           }
```
The `replaceAll` used in that method should be just as effective as the more expensive new method.